cfme/automate/explorer.py

```diff
@@ -391,7 +391,11 @@ def parent(self, p):
 
     def path_str(self):
         """Returns string path to this class, eg ns1/ns2/ThisClass"""
-        return "/".join(self.path)
+        path = "/".join(self.path)
+        if version.current_version() < "5.4":
+            return path
+        else:
+            return "/" + path  # Starts with / from 5.4 onwards because of domains
```

Comment:
yup but in upstream builds, the path is prepended with / whereas in 5.3 it is not. This does not happen in 5.3 so I think it will start to happen in 5.4